spec/presenters/hyrax/object_resource_presenter_spec.rb

```diff
@@ -0,0 +1,9 @@
+# Generated via
+#  `rails generate hyrax:work ObjectResource`
+require 'rails_helper'
+
+RSpec.describe Hyrax::ObjectResourcePresenter do
```

Comment:
Do we need this boilerplate test file? Is this an upstream file we've customized locally and need to test?